pkg/pullrequest/disk.go

```diff
@@ -169,6 +169,26 @@ func FromDisk(path string) (*Resource, error) {
 	var err error
 	var manifest Manifest
 
+	// SCM struct for gitlab pull request does not contain any data that would be written
+	// into head.json or base.json.  Therefore we cannot rely on using that information later
+	// in this method to set top level properties.  Therefore if the pr.json is in the output
+	// dir we will prepopulate r.PR and then update.
+	prPath := filepath.Join(path, "pr.json")
+	_, err = os.Stat(prPath)
+	if os.IsNotExist(err) {
+		// File does not exist, skipping preload data, might cause problems for updating
+		// gitlab statuses
+	} else {
+		b, err := ioutil.ReadFile(path + "/pr.json")
+		if err != nil {
+			return nil, err
+		}
+		err = json.Unmarshal(b, r.PR)
```

Comment:
if err := ...; err != nil {

Reply:
Changed the later of these, the former I cannot wrap as then the bytes will be declared inside the if and unusable in the Unmarshal .... 

Also removed/changed the comment given your change into SCM.